### ISSUE TITLE
Hard Coded Path to Recurrent Flow Completion Model Weight

### ISSUE THREAD

**Issue by ykwongaq** [sczhou/ProPainter#73]
Thank you for the great work on ProPainter!

I am trying to train the `ProPainter` using the pretrained recurrent flow completion model, however, I got a file not found error:

![image](https://github.com/sczhou/ProPainter/assets/54923229/59ec07ab-8ebf-43dc-9001-0717141f3d98)

I found out the root cause is at core/trainer.py line 75:

![image](https://github.com/sczhou/ProPainter/assets/54923229/c435599b-5588-4ad2-943a-6af28b5fd186)

where the path for flow completion model is hard coded.

Perphas it is better to configure it in the train_propainter.json

Thank you

**Comment by Pokerman8** [comment]
I meet the same problem, and I want to know that if the path of the Recurrent Flow Completion Model Weight is `/ProPainter-main/weights/recurrent_flow_completion.pth`

**Comment by ykwongaq** [comment]
> I meet the same problem, and I want to know that if the path of the Recurrent Flow Completion Model Weight is `/ProPainter-main/weights/recurrent_flow_completion.pth`

Yes, I have changed the path to what you mentioned. And it is working now.

**Comment by shenyewei** [comment]
How many hours do you train? I need 1000 hours to train with an A100, is that normal? Thank you.
